src/workspaces/common/WorkspaceMenu.ts

```diff
@@ -160,13 +180,47 @@ const LoadedWorkspaceMenuContent = (props: LoadedWorkspaceMenuContentProps) => {
     [DEFAULT, () => '']
   );
 
+  const menuClicked = (action: string) => {
```

Comment:
@admull

This is what the event will look like for triggering a workspace menu. Origin will be "dashboard" or "list", and "action" will be the menu item triggered.

<img width="797" alt="image" src="https://github.com/user-attachments/assets/9885912d-c653-4cbf-b2f1-5b5c1ca103da">